api/app.py

```diff
@@ -112,6 +113,33 @@ def handle_delete_user():
     return Response(body=None, status_code=204)
 
 
+@app.route("/user/manage_preferences", methods=["POST"])
+def manage_preferences():
+    """Checks for user from email property in request body
+    and sends an email to containing a link to update notification
+    preferences with a temp auth token included
+
+    Returns:
+        [Response]: A formatted response to the API caller
+    """
+    user_email = app.current_request.json_body.get('email')
```

Comment:
I usually opt to validate incoming payloads with a pydantic schema like in the `/user` `POST` endpoint.